notebooks/scripts/causal_estimate_experiments.py

```
@@ -139,6 +139,12 @@
     common_support_threshold=common_support_threshold,
 )
 
+# %%
+true_ate
```

Ruff (0.8.2)

Found useless expression. Either assign it to a variable or remove it.

(B018)